src/parse.yy

```diff
@@ -311,8 +316,14 @@ struct ForeachDimension {
 %type <templateParmList> template_parameter_list template_head
 %type <functionTemplateSym> template_declaration
 
+%precedence LOWER_THAN_ATTRIBUTE
```

Comment:
That's not a clear name. Maybe `STRUCT_WITHOUT_ATTRIBUTE`?

Reply:
Let it be, I have no preferences here.